pkg/cmd/cli/cmd/deploy.go

```diff
@@ -317,7 +322,18 @@ func (o *DeployOptions) cancel(config *deployapi.DeploymentConfig, out io.Writer
 		return fmt.Errorf("couldn't cancel deployment %s", strings.Join(failedCancellations, ", "))
 	}
 	if !anyCancelled {
-		fmt.Fprintln(out, "no active deployments to cancel")
+		if latest == nil {
+			// TODO: this could mean that a new deployment is forthcoming but hasn't
+			// yet been created; might not be worth trying to express in this
+			// context.
+			fmt.Fprintf(out, "no deployments are in progress\n")
```

Comment:
Be more explicit.  No deployments currently in progress for deployment
config foo (current #3)

On Tue, Sep 15, 2015 at 1:20 PM, Dan Mace <notifications@github.com> wrote:

> In pkg/cmd/cli/cmd/deploy.go
> <https://github.com/openshift/origin/pull/4669#discussion_r39538283>:
>
> > @@ -317,7 +322,18 @@ func (o *DeployOptions) cancel(config *deployapi.DeploymentConfig, out io.Writer
> >  		return fmt.Errorf("couldn't cancel deployment %s", strings.Join(failedCancellations, ", "))
> >  	}
> >  	if !anyCancelled {
> > -		fmt.Fprintln(out, "no active deployments to cancel")
> > +		if latest == nil {
> > +			// TODO: this could mean that a new deployment is forthcoming but hasn't
> > +			// yet been created; might not be worth trying to express in this
> > +			// context.
> > +			fmt.Fprintf(out, "no deployments are in progress\n")
>
> The difference in this particular case is there is no latest deployment.
> THere's nothing in progress, but there might be soon- we can't say for sure.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4669/files#r39538283>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift